src/Routes/ImageManager/Images.js

```diff
@@ -75,6 +108,15 @@ const Images = () => {
     loadEdgeImages(dispatch, { limit: perPage, offset: (page - 1) * perPage });
   }, [page, perPage]);
 
+  const handleSort = (_event, index, direction) => {
+    setSortBy({ index, direction });
```

Comment:
handle sort should just update the sort by state and in the `useEffect` check if this value change and if it is then run `loadEdgeImages`.